backend/app/audio.py

```diff
@@ -1,28 +1,28 @@
 """audio processing, etc"""
 from pydub import AudioSegment
+import pandas as pd
+import numpy as np
+import soundfile as sf
 
 
 class Audio:
     silence_threshold = -50.0
-    chunk_size = 10
+    threshold_value = 0.0007  # Tweak the value of threshold to get the aggressive trimming
```

Comment:
What does this threshold value represent? ie what are the units?

Reply:
this is the mask threshold value it will cut the mean value got from rolling windows and discard the value below threshold